change/@fluentui-tokens-a1396b99-e2b7-4ab3-8c89-7d58c5b4ba8b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33393&commitId=9a45d5fedfbcb102d9fe6a6cee18d583fa17653b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.Expand Icon Icon High Contrast.chromium.png|1366| Changed|
|Accordion Converged.Visibility Focus High Contrast.chromium.png|588| Changed|
|Accordion Converged.Disabled High Contrast.chromium.png|1200| Changed|

</details>


  <details><summary><b>Avatar Converged</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Color Active High Contrast.chromium.png|1155| Changed|
|Avatar Converged.Color High Contrast.chromium.png|1155| Changed|
|Avatar Converged.Basic High Contrast.chromium.png|100| Changed|
|Avatar Converged.badgeMask.chromium.png|9| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|

</details>


  <details><summary><b>AvatarGroup Converged</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|AvatarGroup Converged.Basic High Contrast.chromium.png|4730| Changed|
|AvatarGroup Converged.Layout Stack High Contrast.chromium.png|4683| Changed|
|AvatarGroup Converged.Layout Pie 2 High Contrast.chromium.png|1094| Changed|
|AvatarGroup Converged.Overflow Content High Contrast.chromium.png|219| Changed|
|AvatarGroup Converged.Layout Pie 1 High Contrast.chromium.png|1094| Changed|
|AvatarGroup Converged.Overflow Indicator High Contrast.chromium.png|2768| Changed|
|AvatarGroup Converged.Layout Pie High Contrast.chromium.png|1094| Changed|

</details>


  <details><summary><b>Badge Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge Converged.Filled High Contrast.chromium.png|10827| Changed|
|Badge Converged.Ghost High Contrast.chromium.png|7209| Changed|
|Badge Converged.Outline High Contrast.chromium.png|11257| Changed|
|Badge Converged.Tint High Contrast.chromium.png|7914| Changed|

</details>


  <details><summary><b>Breadcrumb Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb Converged.Appearance High Contrast.chromium.png|3030| Changed|

</details>


  <details><summary><b>Button Converged</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Converged.Default High Contrast.chromium.png|479| Changed|
|Button Converged.Subtle High Contrast.chromium.png|402| Changed|
|Button Converged.Outline Disabled High Contrast.chromium.png|315| Changed|
|Button Converged.Subtle Disabled High Contrast.chromium.png|264| Changed|
|Button Converged.Transparent Disabled High Contrast.chromium.png|264| Changed|
|Button Converged.Disabled High Contrast.chromium.png|315| Changed|
|Button Converged.Transparent High Contrast.chromium.png|402| Changed|
|Button Converged.Primary High Contrast.chromium.png|871| Changed|
|Button Converged.Primary Disabled High Contrast.chromium.png|264| Changed|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.Appearance High Contrast.chromium.png|20333| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.Appearance Interactive Filled Alternative High Contrast.chromium.png|12374| Changed|
|Card Converged - Interactive.Appearance Interactive Filled High Contrast.chromium.png|12374| Changed|
|Card Converged - Interactive.Appearance Interactive Outline High Contrast.chromium.png|12254| Changed|
|Card Converged - Interactive.Appearance Interactive Subtle High Contrast.chromium.png|12254| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.Appearance Selectable Outline High Contrast.chromium.png|12254| Changed|
|Card Converged - Selectable.Appearance Selectable Subtle High Contrast.chromium.png|12254| Changed|
|Card Converged - Selectable.Appearance Selectable Filled Alternative High Contrast.chromium.png|12374| Changed|
|Card Converged - Selectable.Appearance Selectable Filled High Contrast.chromium.png|12374| Changed|

</details>


  <details><summary><b>DatePicker Compat</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.Default Hight Contrast.chromium.png|175| Changed|
|DatePicker Compat.Marked Dates Hight Contrast.chromium.png|1878| Changed|
|DatePicker Compat.Date Range Month Hight Contrast.chromium.png|1878| Changed|
|DatePicker Compat.Show Week Numbers Hight Contrast.chromium.png|1960| Changed|

</details>


  <details><summary><b>Dialog</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Default High Contrast.chromium.png|22010| Changed|
|Dialog.Nested High Contrast.chromium.png|3544| Changed|
|Dialog.Alert High Contrast.chromium.png|22010| Changed|
|Dialog.Actions Position Start Position End High Contrast.chromium.png|23262| Changed|
|Dialog.No Actions High Contrast.chromium.png|17483| Changed|
|Dialog.Actions Position Start High Contrast.chromium.png|22001| Changed|
|Dialog.No Title High Contrast.chromium.png|20678| Changed|
|Dialog.No Title No Actions High Contrast.chromium.png|16151| Changed|
|Dialog.Title Custom Action High Contrast.chromium.png|18250| Changed|
|Dialog.Scroll Long Content High Contrast.chromium.png|79153| Changed|
|Dialog.Non Modal High Contrast.chromium.png|25737| Changed|

</details>


  <details><summary><b>Divider Converged - Horizontal</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider Converged - Horizontal.Appearance Subtle High Contrast.chromium.png|95| Changed|
|Divider Converged - Horizontal.Appearance Strong High Contrast.chromium.png|95| Changed|
|Divider Converged - Horizontal.Appearance Brand High Contrast.chromium.png|95| Changed|
|Divider Converged - Horizontal.With Content High Contrast.chromium.png|95| Changed|

</details>


  <details><summary><b>Drawer</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Inline High Contrast.chromium.png|2005| Changed|
|Drawer.Default High Contrast.chromium.png|1684| Changed|
|Drawer.Alert Overlay High Contrast.chromium.png|1684| Changed|
|Drawer.Large Content Scroll Overlay High Contrast.chromium.png|1684| Changed|
|Drawer.Large Content Scroll Inline High Contrast.chromium.png|20606| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|3226| Changed|
|Drawer.Inline High Contrast.chromium.png|2005| Changed|
|Drawer.Large Overlay High Contrast.chromium.png|1684| Changed|
|Drawer.Large Inline High Contrast.chromium.png|2005| Changed|
|Drawer.Medium Inline High Contrast.chromium.png|2005| Changed|
|Drawer.Overlay High Contrast.chromium.png|1684| Changed|
|Drawer.Position End Inline High Contrast.chromium.png|6682| Changed|
|Drawer.Non Modal Overlay High Contrast.chromium.png|1684| Changed|
|Drawer.Medium Overlay High Contrast.chromium.png|1684| Changed|
|Drawer.Position End Overlay High Contrast.chromium.png|1684| Changed|
|Drawer.Separator Position Start Inline High Contrast.chromium.png|6682| Changed|
|Drawer.overlay drawer full.chromium.png|1138| Changed|
|Drawer.Position Start Overlay High Contrast.chromium.png|1684| Changed|
|Drawer.Position Start Inline High Contrast.chromium.png|6682| Changed|
|Drawer.Separator Position End Inline High Contrast.chromium.png|6682| Changed|

</details>


  <details><summary><b>InfoLabel</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.Default High Contrast.chromium.png|497| Changed|

</details>


  <details><summary><b>InteractionTag Converged</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InteractionTag Converged.Default High Contrast.chromium.png|365| Changed|
|InteractionTag Converged.Brand Disabled High Contrast.chromium.png|314| Changed|
|InteractionTag Converged.Disabled High Contrast.chromium.png|314| Changed|
|InteractionTag Converged.Brand High Contrast.chromium.png|478| Changed|
|InteractionTag Converged.Filled High Contrast.chromium.png|478| Changed|
|InteractionTag Converged.Outline High Contrast.chromium.png|478| Changed|
|InteractionTag Converged.Outline Disabled High Contrast.chromium.png|314| Changed|

</details>


  <details><summary><b>Label Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label Converged.Disabled High Contrast.chromium.png|313| Changed|
|Label Converged.Required High Contrast.chromium.png|476| Changed|
|Label Converged.Root High Contrast.chromium.png|247| Changed|

</details>


  <details><summary><b>Link Converged - Rendered as anchor</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as anchor.Inverted High Contrast.chromium.png|677| Changed|
|Link Converged - Rendered as anchor.Inverted Disabled High Contrast.chromium.png|601| Changed|
|Link Converged - Rendered as anchor.Stand Alone Disabled Focusable High Contrast.chromium.png|558| Changed|
|Link Converged - Rendered as anchor.Stand Alone High Contrast.chromium.png|329| Changed|

</details>


  <details><summary><b>Link Converged - Rendered as button</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as button.Inverted Disabled High Contrast.chromium.png|601| Changed|
|Link Converged - Rendered as button.Inverted High Contrast.chromium.png|677| Changed|

</details>


  <details><summary><b>Link Converged - Rendered as span</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as span.Inverted Disabled High Contrast.chromium.png|601| Changed|
|Link Converged - Rendered as span.Inverted High Contrast.chromium.png|677| Changed|

</details>


  <details><summary><b>Menu Converged - groups</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - groups.Default High Contrast.chromium.png|1049| Changed|

</details>


  <details><summary><b>Menu Converged - selection</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection.Checkbox High Contrast.chromium.png|175| Changed|
|Menu Converged - selection.Switch High Contrast.chromium.png|1498| Changed|

</details>


  <details><summary><b>MessageBar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Intents High Contrast.chromium.png|8649| Changed|

</details>


  <details><summary><b>Persona Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona Converged.Basic High Contrast.chromium.png|1351| Changed|

</details>


  <details><summary><b>Radio Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio Converged.Disabled Checked High Contrast.chromium.png|273| Changed|
|Radio Converged.Disabled High Contrast.chromium.png|143| Changed|
|Radio Converged.Checked High Contrast.chromium.png|168| Changed|
|Radio Converged.Unchecked High Contrast.chromium.png|259| Changed|

</details>


  <details><summary><b>RatingDisplay Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RatingDisplay Converged.Brand With Half Value High Contrast.chromium.png|70| Changed|
|RatingDisplay Converged.Neutral With Half Value High Contrast.chromium.png|70| Changed|
|RatingDisplay Converged.Marigold With Half Value High Contrast.chromium.png|70| Changed|

</details>


  <details><summary><b>SpinButton Converged</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.Appearance Outline Default High Contrast.chromium.png|142| Changed|
|SpinButton Converged.Appearance Filled Lighter High Contrast.chromium.png|115| Changed|
|SpinButton Converged.Appearance Underline High Contrast.chromium.png|84| Changed|
|SpinButton Converged.At Min Bound High Contrast.chromium.png|137| Changed|
|SpinButton Converged.Appearance Filled Darker High Contrast.chromium.png|142| Changed|
|SpinButton Converged.Custom Width High Contrast.chromium.png|59| Changed|
|SpinButton Converged.At Max Bound High Contrast.chromium.png|137| Changed|
|SpinButton Converged.Display Value High Contrast.chromium.png|264| Changed|
|SpinButton Converged.Size Small High Contrast.chromium.png|43| Changed|
|SpinButton Converged.Disabled High Contrast.chromium.png|93| Changed|

</details>


  <details><summary><b>Spinner converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Primary With Label High Contrast.chromium.png|1450| Changed|
|Spinner converged.Inverted With Label High Contrast.chromium.png|1450| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Default High Contrast.chromium.png|1198| Changed|

</details>


  <details><summary><b>Switch Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch Converged.Disabled And Checked High Contrast.chromium.png|540| Changed|
|Switch Converged.Disabled And Unchecked High Contrast.chromium.png|540| Changed|
|Switch Converged.Enabled And Checked High Contrast.chromium.png|636| Changed|
|Switch Converged.Enabled And Unchecked High Contrast.chromium.png|636| Changed|

</details>


  <details><summary><b>TabList and Tab Converged</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TabList and Tab Converged.Default High Contrast.chromium.png|485| Changed|
|TabList and Tab Converged.Filled Circular Appearance High Contrast.chromium.png|775| Changed|
|TabList and Tab Converged.Large Size High Contrast.chromium.png|640| Changed|
|TabList and Tab Converged.Small Size High Contrast.chromium.png|485| Changed|
|TabList and Tab Converged.Tab Selected Default High Contrast.chromium.png|593| Changed|
|TabList and Tab Converged.Vertical And Small Size High Contrast.chromium.png|420| Changed|
|TabList and Tab Converged.Subtle Circular Appearance High Contrast.chromium.png|1105| Changed|
|TabList and Tab Converged.Subtle Appearance High Contrast.chromium.png|485| Changed|
|TabList and Tab Converged.Tab Selected High Contrast.chromium.png|593| Changed|
|TabList and Tab Converged.Vertical And Large Size High Contrast.chromium.png|560| Changed|

</details>


  <details><summary><b>Table layout flex</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex.Multiselect Checked Neutral High Contrast.chromium.png|5312| Changed|
|Table layout flex.Multiselect Checked Brand High Contrast.chromium.png|7282| Changed|
|Table layout flex.Single Select Checked Neutral High Contrast.chromium.png|6671| Changed|
|Table layout flex.Multiselect High Contrast Story.chromium.png|7282| Changed|
|Table layout flex.Multiselect Mixed Neutral High Contrast.chromium.png|6190| Changed|
|Table layout flex.Size Medium High Contrast Story.chromium.png|7282| Changed|
|Table layout flex.Multiselect Mixed Brand High Contrast.chromium.png|7282| Changed|
|Table layout flex.Single Select Checked Brand High Contrast.chromium.png|7282| Changed|
|Table layout flex.Single Select Checked High Contrast Story.chromium.png|7282| Changed|
|Table layout flex.Single Select High Contrast Story.chromium.png|7282| Changed|

</details>


  <details><summary><b>Table layout flex - cell actions</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - cell actions.Always Visible High Contrast.chromium.png|7282| Changed|
|Table layout flex - cell actions.Default High Contrast.chromium.png|7282| Changed|

</details>


  <details><summary><b>Table layout flex - headers</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - headers.Sortable High Contrast.chromium.png|7346| Changed|

</details>


  <details><summary><b>Table layout table</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table.Multiselect Checked Neutral High Contrast.chromium.png|5312| Changed|
|Table layout table.Multiselect High Contrast Story.chromium.png|7282| Changed|
|Table layout table.Multiselect Checked Brand High Contrast.chromium.png|7282| Changed|
|Table layout table.Multiselect Mixed Brand High Contrast.chromium.png|7282| Changed|
|Table layout table.Single Select Checked Brand High Contrast.chromium.png|7282| Changed|
|Table layout table.Multiselect Mixed Neutral High Contrast.chromium.png|6190| Changed|
|Table layout table.Single Select Checked Neutral High Contrast.chromium.png|6671| Changed|
|Table layout table.Single Select Checked High Contrast Story.chromium.png|7282| Changed|
|Table layout table.Size Medium High Contrast Story.chromium.png|7282| Changed|
|Table layout table.Single Select High Contrast Story.chromium.png|7282| Changed|

</details>


  <details><summary><b>Table layout table - cell actions</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - cell actions.Always Visible High Contrast.chromium.png|7282| Changed|
|Table layout table - cell actions.Default High Contrast.chromium.png|7282| Changed|

</details>


  <details><summary><b>Table layout table - headers</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - headers.Sortable High Contrast.chromium.png|7346| Changed|

</details>


  <details><summary><b>Tag Converged</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tag Converged.Brand Disabled High Contrast.chromium.png|274| Changed|
|Tag Converged.Brand High Contrast.chromium.png|417| Changed|
|Tag Converged.Default High Contrast.chromium.png|365| Changed|
|Tag Converged.Disabled High Contrast.chromium.png|274| Changed|
|Tag Converged.Filled High Contrast.chromium.png|417| Changed|
|Tag Converged.Outline High Contrast.chromium.png|417| Changed|
|Tag Converged.Outline Disabled High Contrast.chromium.png|274| Changed|

</details>


  <details><summary><b>TagPicker</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.Default Open High Contrast.chromium.png|5619| Changed|
|TagPicker.Disabled High Contrast.chromium.png|1673| Changed|
|TagPicker.Expand Icon High Contrast.chromium.png|242| Changed|
|TagPicker.Appearance High Contrast.chromium.png|6355| Changed|
|TagPicker.Grouped High Contrast.chromium.png|5619| Changed|
|TagPicker.Size High Contrast.chromium.png|798| Changed|
|TagPicker.Secondary Action High Contrast.chromium.png|386| Changed|

</details>


  <details><summary><b>Text Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text Converged.Default High Contrast.chromium.png|26567| Changed|

</details>


  <details><summary><b>ToggleButton Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ToggleButton Converged.Default High Contrast.chromium.png|479| Changed|
|ToggleButton Converged.Primary High Contrast.chromium.png|871| Changed|

</details>


  <details><summary><b>Tooltip Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip Converged.With Arrow High Contrast.chromium.png|516| Changed|
|Tooltip Converged.Basic High Contrast.chromium.png|374| Changed|

</details>


  <details><summary><b>Tree</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.Actions High Contrast.chromium.png|648| Changed|
|Tree.Default High Contrast.chromium.png|972| Changed|
|Tree.Default Open Tree High Contrast.chromium.png|2966| Changed|
|Tree.Flat Tree Single Selection High Contrast.chromium.png|353| Changed|
|Tree.Appearance High Contrast.chromium.png|1319| Changed|
|Tree.Flat High Contrast.chromium.png|267| Changed|
|Tree.Persona High Contrast.chromium.png|3855| Changed|
|Tree.Expand Icon High Contrast.chromium.png|1966| Changed|
|Tree.Layout High Contrast.chromium.png|670| Changed|
|Tree.Flat Tree Multi Selection High Contrast.chromium.png|353| Changed|
|Tree.Size High Contrast.chromium.png|2484| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>